src/test/util.ts

```diff
@@ -47,6 +51,247 @@ export function createFileSynchronizer(baseDir?: string, debugging?: boolean) {
   return {synchronizer, serverConnection, clientConnection, baseDir, converter};
 }
 
+export async function createTestEnvironment(
```

Comment:
could you add documentation or an example on how this function & classes are expected to be used?